distribution/src/bin/elasticsearch

```diff
@@ -45,6 +50,15 @@ then
   fi
 fi
 
+if [[ $AUTO_CONFIG = true ]]; then
+  ES_MAIN_CLASS=org.elasticsearch.xpack.security.cli.AutoConfigInitialNode \
+    ES_ADDITIONAL_SOURCES="x-pack-env;x-pack-security-env" \
+    ES_ADDITIONAL_CLASSPATH_DIRECTORIES=lib/tools/security-cli \
+    "`dirname "$0"`"/elasticsearch-cli \
```

Comment:
Just a suggestion - this maybe reads a little clearer?

```suggestion
    "$(dirname "$0")/elasticsearch-cli" \
```

Reply:
Thank you Rory!